python/instrumentation/openinference-instrumentation-bedrock/README.md

```diff
@@ -1,9 +1,71 @@
 # OpenInference AWS Bedrock Instrumentation
 
+Python autoinstrumentation library for AWS Bedrock made using `boto3`.
+
+This package implements OpenInference tracing for `invoke_model` calls made using a `boto3` `bedrock-runtime` client. These traces are fully OpenTelemetry compatible and can be sent to an OpenTelemetry collector for viewing, such as [Arize `phoenix`](https://github.com/Arize-ai/phoenix).
+
 [![pypi](https://badge.fury.io/py/openinference-instrumentation-bedrock.svg)](https://pypi.org/project/openinference-instrumentation-bedrock/)
 
 ## Installation
 
 ```shell
 pip install openinference-instrumentation-bedrock
 ```
+
+## Quickstart
+
+Install `openinference-instrumentation-bedrock`, `arize-phoenix` and `boto3`.
```

Comment:
Maybe mention that this should be run in a notebook. you could also add a colab link here!